rskj-core/src/fuzzTest/java/co/rsk/util/RLPFuzzTest.java

```
@@ -0,0 +1,459 @@
+package co.rsk.util;
+
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+import com.code_intelligence.jazzer.junit.FuzzTest;
+import org.bouncycastle.util.BigIntegers;
+import org.ethereum.db.ByteArrayWrapper;
+import org.ethereum.util.RLP;
+import org.ethereum.util.RLPElement;
+import org.ethereum.util.RLPList;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Tag;
+
+import java.math.BigInteger;
+import java.util.*;
+
+// JAZZER_FUZZ=1 ./gradlew fuzzTest --tests co.rsk.util.RLPFuzzTest.fuzzDecodeInt
+class RLPFuzzTest {
+    @Tag("RLPFuzzDecodeInt")
+    @FuzzTest
+    public void fuzzDecodeInt(FuzzedDataProvider data) {
+        byte[] input = generateMixedRLPData(data);
+        if (input.length < 1) { return; }
+        // int index = data.consumeInt(0, input.length - 1);
+        int index = 0;
+        try {
+            RLP.decodeInt(input, index);
+        } catch (RLPException e) {
+            
```

I don't like this empty catch block here